README.rst

```diff
@@ -99,7 +99,7 @@ required).
     mlx-warnings --command <yourcommand>
 
 ---------------
-Running command
+Running Command
```

Comment:
why would you have a capitalized second word? (generally speaking)

Reply:
Because it's the correct format. FYI https://www.webucator.com/how-to/how-capitalize-headings-titles.cfm